tableau_exporter/server_parser_status_metrics.py

```diff
@@ -0,0 +1,66 @@
+from prometheus_client.core import GaugeMetricFamily
+
+
+class TableauServerStatusParser(object):
+    DEFAULT_LABEL_KEYS = ['machine', 'process', 'status']
+
+    STATUS_MAP = {
+        'Active': 0,
```

Comment:
This is a initialisation value for a mutable state. Keeping it in class reduces readability.